src/ace.F90

```diff
@@ -378,8 +379,8 @@ subroutine read_ace_table(i_table, i_listing)
       if (data_0K) then
         continue
       else
-        call read_nu_data(nuc)
         call read_reactions(nuc)
+        call read_nu_data(nuc)
```

Comment:
Yeah I agree. That may beat wait until the transition to h5 data is
complete anyways
On Mar 21, 2016 11:52 AM, "Paul Romano" <notifications@github.com> wrote:

> In src/ace.F90
> <https://github.com/mit-crpg/openmc/pull/610#discussion_r56845766>:
>
> >          call read_reactions(nuc)
> > +        call read_nu_data(nuc)
>
> Ditto here -- I think I'll do a separate PR moving around routines to
> their proper places if you're ok with it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/openmc/pull/610/files/9304b3806120305dac6aa59133db233a070528f5#r56845766>
>